e2e/common/traits/health_test.go

```diff
@@ -29,11 +29,8 @@ import (
 	"testing"
 	"time"
 
-	. "github.com/onsi/gomega"
```

Comment:
I think these removals might be the source of the IT's failure. You need to add them again.